src/api/fileApi.ts

```diff
@@ -1,14 +1,21 @@
 import { AwsS3Part } from '@uppy/aws-s3-multipart';
 import { UppyFile } from '@uppy/core';
 import { isSuccessStatus } from '../utils/constants';
-import { FileApiPath } from './apiPaths';
+import { FileApiPath, PublicationsApiPath } from './apiPaths';
 import { apiRequest, authenticatedApiRequest, authenticatedApiRequest2 } from './apiRequest';
 
 interface DownloadFileResponse {
   id: string;
-  expires: string;
+  uri?: string;
```

Comment:
Har snakket med @truhacevkir om å samkjøre responsen for å laste ned fil for registrering og importkandidat. Venter til det er løst og det er fikset her også før jeg setter PR til ready for review

Reply:
Solved!